src/Router.jsx

```diff
@@ -40,11 +52,15 @@ export const Router = () => {
 
                     <Route
                         exact
-                        path={`${routes.Profile}/:id`}
-                        element={<ProfilePage />}
+                        path={`${routes.Post}/:id`}
+                        element={<PostDetailsPage />}
```

Comment:
why we don't have route to profile?

Reply:
sorry, accidentally removed during cleanup